packages/bpk-docs/src/themableAttributes.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable global-require */
+
+export default [
+  require('bpk-component-blockquote').themeAttributes,
```

Comment:
Why are we using require here? just import them like normal

Reply:
well doing it like this I can set them into an array and filter straight off, importing them I have to import them as alias, create the array and filter, I think it's much more readable this way, wdyt?